server/app/utils.py

```diff
@@ -319,9 +319,13 @@ def add_to_grading_queues(assign_key, cursor=None, num_updated=0):
     seen = set()
     for queue in queues:
         for subm in queue.submissions:
-            seen.add(subm.get().submitter.id())
+            try:
```

Comment:
Fixed. The app was whining about a circular import, so I had to test for `ndb.Key` instead.